### PR TITLE
Bugfix for find command

### LINKED ISSUE #114: Fix issue with spaces in find command
When using the default delimiter (semicolon), spaces between items like name and courses in find command causes output of parser to miss out on some items.

### PR DESCRIPTION
**Pull request by Clarenceeey**
Fixes #114 

- Find command now parses input correctly, even if there are spaces between them
- Added test cases to test FindCommandParser

### CODE REVIEW

**Review by yhanyi — APPROVED** [review]
LGTM, nice job using ArgumentTokenizer in FindComandParser.

File: src/main/java/seedu/address/logic/parser/FindCommandParser.java [hunk 1]
```diff
@@ -57,11 +57,11 @@ private void combinePredicates(ArgumentMultimap argMultimap,
             if (prefix.equals(PREFIX_NAME)) {
                 argMultimap.getAllValues(prefix)
                         .forEach(pre -> predicateList.add(
-                                new NameContainsKeywordsPredicate(List.of(pre.split(";")))));
+                                new NameContainsKeywordsPredicate(ArgumentTokenizer.tokenizeWithDefault(pre))));
```
Comment (notnotmax) [inline src/main/java/seedu/address/logic/parser/FindCommandParser.java:60]: Good use of your method here.

File: src/test/java/seedu/address/logic/parser/FindCommandParserTest.java
Comment (notnotmax) [inline src/test/java/seedu/address/logic/parser/FindCommandParserTest.java:1]: Looks a bit wordier but well done making the code more maintainable with constants.

**Review by notnotmax — APPROVED** [review]
lgtm, ready for merge.

File: src/test/java/seedu/address/logic/parser/FindCommandParserTest.java [hunk 2]
```diff
@@ -20,76 +32,106 @@ public class FindCommandParserTest {
...
+                + SPACED_PREFIX_COURSE + " " + VALID_COURSE_LOWERCASE_CS2030 + DEFAULT_DELIMITER
+                + VALID_COURSE_CS2101 + " " + DEFAULT_DELIMITER
+                + " " + VALID_COURSE_CS2103T + " ", expectedFindCommand);
+
+        // line breaks and tabs between keywords
+        assertParseSuccess(parser, SPACED_PREFIX_NAME + "\n" + VALID_NAME_AMY + "\t" + DEFAULT_DELIMITER
+                + "\t" + VALID_NAME_BOB + "\t"
+                + SPACED_PREFIX_COURSE + " " + VALID_COURSE_CS2030 + "\n" + DEFAULT_DELIMITER
+                + "\t " + VALID_COURSE_CS2101 + DEFAULT_DELIMITER
+                + " " + VALID_COURSE_CS2103T + "\t", expectedFindCommand);
```
Comment (S-K-Y-Light) [inline src/test/java/seedu/address/logic/parser/FindCommandParserTest.java:135]: Nice Test Cases. Very Extensive.

File: src/main/java/seedu/address/logic/parser/FindCommandParser.java (hunk 1 above)
Comment (S-K-Y-Light) [inline src/main/java/seedu/address/logic/parser/FindCommandParser.java:60]: Agree with @notnotmax. Good Catch to improve Code Quality.

**Review by S-K-Y-Light — COMMENTED** [review]
LGTM.

**Pull request merged**